proposals/10-egress-gateways.md

```diff
@@ -0,0 +1,76 @@
+# Egress Gateways
+
+* Authors: @shaneutt
+* Status: Proposed
+
+# What?
+
+Provide standards in Kubernetes to route traffic outside of the cluster.
+
+# Why?
+
+Applications are increasingly utilizing inference as a part of their logic.
+This may be for chatbots, knowledgebases, or a variety of other potential use
+cases. The inference workloads to support this may not always be on the same
+cluster as the requesting workload. Inference workloads may be on a separate
+cluster, as the organization centralizes them, or they may be located
+specifically for reasons of regionality. Even then, not all organizations are
+going to run inference workloads themselves, and will utilize 3rd party cloud
+services. All of this points to a need to provide standards for how Kubernetes
+workloads reach these external inference sources, and provide the same AI
+Gateway security, control and management capabilities that are required for the
+ingress use case.
+
+## User Stories
+
+* As a cluster admin I need to provide inference to workloads on my cluster,
+  but I provide a dedicated cluster for this so that I can manage it
+  separately.
```

Comment:
This definitely overlaps the most with existing Gateway API Inference Extension scope, specifically https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1374 (worth reading linked design doc too for alternatives considered and potential future alternative approaches)